index.html

```diff
@@ -0,0 +1,179 @@
+                    <h3>Plot 60, Independence Layout Enugu, Biafra.</h3>
+                </div>
+            </div>
+        </section>
+
+        <section id="program">
+            <div class="program-title">
+                <h2>Why use our services</h2>
+                <hr>
+            </div>
+            <div class="wrapper program-content">
+                <div class="program-card">
+                    <div class="card-icon">
+                        <img src="images/icons/icon1.png" alt="icon 1">
+                        <h3>Good quality materials</h3>
+                    </div>
+                    <div class="card-description">
+                        <p>We provide one of the best qualities in the town.</p>
+                    </div>
+                </div>
+                <div class="program-card">
+                    <div class="card-icon">
+                        <img src="images/icons/icon2.svg" alt="icon 2">
+                        <h3>Professionalism</h3>
+                    </div>
+                    <div class="card-description">
+                        <p>We are dedicated to customers satisfaction</p>
+                    </div>
+                </div>
+                <div class="program-card">
+                    <div class="card-icon">
+                        <img src="images/icons/icon3.png" alt="icon 3">
+                        <h3>Perfect precision </h3>
+                    </div>
+                    <div class="card-description">
+                        <p>We are always untop of the game, when it comes to service delivery</p>
+                    </div>
+                </div>
+                <div class="program-card">
+                    <div class="card-icon">
+                        <img src="images/icons/icon4.png" alt="icon 4">
+                        <h3>Modern design</h3>
+                    </div>
+                    <div class="card-description">
+                        <p>We are sensitive to trends and we keep up with current innovations</p>
+                    </div>
+                </div>
+                <div class="program-card">
+                    <div class="card-icon">
+                        <img src="images/icons/icon5.png" alt="icon 5">
+                        <h3>Featured in magazines</h3>
+                    </div>
+                    <div class="card-description">
+                        <p>We have created a lot of websites for several businesses</p>
+                    </div>
+                </div>
+            </div>
+            <div class="program-btn">
+                <button type="button">Join us at 3ple Tech for your software development</button>
+            </div>
+        </section>
+
+    </main>
+
+    <section id="testimonials">
+        <div class="testimonial-title">
+          <h2 class="testimonial-header">Testimonials</h2>
+          <hr>
+        </div>  
+        <div class="testimonial-container">
```

Comment:
[OPTIONAL] I suggest you change this to a `<section>` since it's the correct semantic tag. Its always important to use the correct tag so that screen readers understand your page. Note that though not highlighted by the previous reviewer this is a core requirement.